sdk/monitor/Azure.Monitor.Query/tests/LogsTestData.cs

```diff
@@ -42,7 +42,7 @@ public class LogsTestData
 
         private string TableANameSent => nameof(TableA) + DataVersion + "_" + RetentionWindowStart.DayOfYear;
         public string TableAName => TableANameSent + "_CL";
-        public QueryTimeRange DataTimeRange => new QueryTimeRange(RetentionWindowStart, TimeSpan.FromDays(7));
+        public QueryTimeRange DataTimeRange => new QueryTimeRange(RetentionWindowStart, TimeSpan.FromDays(14));
```

Comment:
This range is set to calculate 7 days from the last Monday which doesn't include today's date. Expanding this range to include today's date in the calculation

Reply:
Looking at the failure it seems you need to rerecord your tests.

Reply:
I think we do need to re-record some tests but the live test pipeline is failing due to another issue.

Reply:
Yeah but we will not get to test the live test pipelines with these changes until we either check these changes in or trigger the live test manually against your private branch.

Reply:
It might be best to do both actually. Re-record the tests so the CI pipelines get green and trigger the live test pipelines to test your changes against live pipelines.